inference-engine/tests/functional/plugin/conformance/test_runner/skip_configs/skip_config_cpu.lst

```diff
@@ -0,0 +1,4 @@
+# Hung:
+.*AvgPool_1199829.*
```

Comment:
do we have such list for `TEMPLATE` plugin?

Reply:
Fully agree. Added. Thanks!